Init codeowners (#29)
IBM/cloudant-python-sdk: CONTRIBUTING.md

> # Contributing

Author: I reverted it in 8ed263f.

## Questions

If you are having problems using the APIs or have a question about IBM Cloud
services, please ask a question at
[Stack Overflow](http://stackoverflow.com/questions/ask?tags=ibm-cloud).

## Issues

If you encounter an issue with the project, you are welcome to submit a [bug
report](https://github.com/IBM/cloudant-python-sdk/issues). Before that, please
search for similar issues. It's possible that someone has already reported the
problem.

## PRs

The code in the repository is partly generated from the Cloudant OpenAPI
specification, so it is owned by @cloudant-sdks-automation. This means PRs to
code, tests and sometimes even markdown are often inappropriate and we may need
to make changes to the specification instead. Please, first [look at the list
of files](.github/CODEOWNERS) that _should not_ be modified. If you still
want to modify one of the listed files we ask that an **issue is always opened
prior to a PR** to give us the opportunity to discuss the best place for the
change before investing your effort on a patch that we may not be able to

> accept.

Author: Thanks, I was not sure about my sentences as well, this is a much nicer served paragraph. Fixed in 9ec1982.

Before that, please search for similar issues. It's possible that someone has already reported the problem.

# PRs

> We ask that an **issue is always opened prior to a PR** to give us the

Author: Since the sentence is starting in plural first person and tries to be informal here, grammatically I do not see this modification as reasonable.

Maybe if the whole sentence would be formal, but I like the informal version better:
```suggestion
The Cloudant team asks that an **issue is always opened prior to a PR** to give them the
```

> We ask that an **issue is always opened prior to a PR** to give us the

Reviewer: No need to change, let's keep as-is.

opportunity to discuss the best place for the change before investing
your effort on a patch that we may not be able to accept.

> The code in the repository is partly generated from the Cloudant OpenAPI
> specification. This means PRs to code, tests and sometimes even markdown
> are often inappropriate and we may need to make changes to the specification
> instead. Please, first [look at the list of files](.github/CODEOWNERS) that
> _should not_ be modified.

Reviewer: We should remove `partly` as it opens the door for questions like "which parts?"
I've also suggested some other changes:
```suggestion
This repository is generated from the Cloudant OpenAPI
specification. PRs to source code, tests and markdown documentation
might be improper if the change is required within the OpenAPI specification.
Please [review the list of files](.github/CODEOWNERS) that _should not_ be modified.
```

Author: For me, it is clear the `partly` refers to the list of files mentioned in one of the later sentences. That is why I would keep the first sentence as-is, mainly because not the whole repository, but a part of it is generated. However, I fancy the simplification of the second sentence.

What do you think about this:
```suggestion
The code in the repository is partly generated from the Cloudant OpenAPI
specification. PRs to source code, tests, and markdown
might be improper if the change is required within the OpenAPI specification. 
Please, first [look at the list of files](.github/CODEOWNERS) that
_should not_ be modified.
```

Reviewer: No need to change, keep as-is.

> _should not_ be modified.

Reviewer 2: `Please, first [look at the list of files](.github/CODEOWNERS) that
_should not_ be modified.`

I think this statement invalid.

The codeowner setting does not deny the modification of files, but notifies the team about a PR is opened and requires a review before merge.

> Code owners are automatically requested for review when someone opens a pull request that modifies code that they own.
Source: https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners

Reviewer 2: I think we should collect rather which files can be modified by contributors and in which area we expect extensions.
